src/mudblazor/MudBlazor.Template.Client/Program.cs

```diff
@@ -7,7 +7,6 @@
 
 var builder = WebAssemblyHostBuilder.CreateDefault(args);
 
-builder.Services.AddScoped(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
```

Comment:
I'm not sure what this line was for?   Templates that I tested didn't seem to need it.

Reply:
I think this is part of the [default .NET WASM template](https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Program.cs#L9).

Reply:
I think we can leave this line as it's standard one, it's so that the client could make backed requests in case you co host with asp.net core server and they are on the same url

Reply:
Ok, thanks for explaining!  I've added the line back in.